lib/install/management/virtual_app.go

```diff
@@ -79,7 +79,7 @@ func (d *Dispatcher) createVApp(conf *config.VirtualContainerHostConfigSpec, set
 			Name:      "vSphere Integrated Containers",
 			Vendor:    "VMware",
 			VendorUrl: "http://www.vmware.com/",
-			Version:   "0.0.1",
+			Version:   "0.5.5",
```

Comment:
I agree about the version package, however, it currently doesn't show the correct version. Right now it shows as 0.4.0, so you might not always have the most up-to-date version by using the package.